powerline/bindings/lemonbar/powerline-lemonbar-multimonitor.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+import sys
+import time
+import re
+import subprocess
+
+from threading import Lock, Timer
+from argparse import ArgumentParser, REMAINDER
+
+from powerline.lemonbar import LemonbarPowerline
+from powerline.lib.monotonic import monotonic
+from powerline.lib.dict import mergedicts
+
+
+if __name__ == '__main__':
+	parser = ArgumentParser(
+		description='Powerline BAR bindings.',
+		usage='%(prog)s [-h] [--i3] [--height HEIGHT] [--theme THEME [THEME ...]] -- args')
+	parser.add_argument(
+		'--i3', action='store_true',
+		help='Subscribe for i3 events.'
+	)
+	parser.add_argument(
+		'--height', default='',
+		help='Bar height'
+	)
+	parser.add_argument(
+		'--theme', '-t',
+		default=[], nargs='+',
+		help='makes all outputs that containing a string use the specified theme. Format like "out=theme"'
```

Comment:
Inconsistent documentation, missing sentence end. Also I will not accept this as well, use overrides. If you need output-specific themes then this is needed to be a local theme (e.g. `"local_themes": {"{output}": "{theme_name}"}` in `config.json/ext/wm`) like you may see in vim and shell bindings, *not* `--theme output=theme` argument.